boltdesignsystem/bolt
# DS-541 Floating Action Buttons

## Jira

- [DS-541](https://pegadigitalit.atlassian.net/browse/DS-541)
- [DS-540](https://pegadigitalit.atlassian.net/browse/DS-540)

## Summary

Create a component for floating action buttons.

## Details

1. Added the new Floating Action Buttons component
2. Created full page demos to showcase the component usage
3. Fixed a bug in Button element where icon is way too big for small icon-only buttons

## How to test

1. Pull Down the Branch and update.
2. Check out the demo pages.

## Release notes

New component named Floating Action Buttons added.

### Visual changes

Icon-only button with size set to `small` renders smaller icon now. Overall button size is not affected.
